.github/workflows/__build-workflow.yaml

```diff
@@ -208,6 +208,9 @@ jobs:
       - name: Build image
         id: build
         uses: docker/build-push-action@v6.3.0
+        env:
+          DOCKER_BUILD_RECORD_UPLOAD: false
+          DOCKER_BUILD_SUMMARY: false
```

Comment:
I would still keep the `DOCKER_BUILD_SUMMARY=true` to upload summary as GH job summay where the inputs are masked / redacted on github. Ex: https://github.com/Kong/gateway-operator/actions/runs/9885055008.(Refer `Build inputs` on Docker summary for each job)

Reply:
Discussed in slack.

Reply:
Agree to disable this as it  will expose the summary which shows that we're using the token which might already be an inclination to folks scavenging across the :github:. However i would have the team enable this summary when the secrets are used to pass the PAT instead of docker args.